must-gather/collection-scripts/gather_common_ceph_resources

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
```

Comment:
Can't you do `BASE_COLLECTION_PATH="${$1:-pwd}`?

Reply:
This will invalidate https://github.com/openshift/ocs-operator/blob/master/must-gather/collection-scripts/gather_ceph_resources#L8-L11

Reply:
How? This would make it so that if `$1` is not specified just use the output `pwd`. `gather_ceph_resources` will always call it with `BASE_COLLECTION_PATH` as `$1`, so there's no need for the `if` later on in this file.

Reply:
@jarrpa the suggestion is correct but we are doing this way in all must-gather scripts. It should be changed to the suggested method in one separate commit. Does it sounds good.